<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaciloRacuna</class>
 <widget class="QDialog" name="PlaciloRacuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>490</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>494</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
    <italic>true</italic>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>PLAČILO</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="0">
       <widget class="QRadioButton" name="radioButton_placiloSkupaj">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Znesek plačila (celotno plačilo)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="lineEdit">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QRadioButton" name="radioButton_placiloEnkratno">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Znesek plačila (obročno plačilo)</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QDateEdit" name="dateEdit">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_2">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/icons/bin.png</normaloff>:/icons/bin.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="files.qrc">
            <normaloff>:/icons/fix.png</normaloff>:/icons/fix.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>22</width>
            <height>22</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLineEdit" name="lineEdit_TrenutnoPlacilo">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLineEdit" name="lineEdit_opombe">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Terjatev / obveznost</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_terObv">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>16</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Datum plačila</string>
        </property>
        <property name="buddy">
         <cstring>dateEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Vnos</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QPushButton" name="pushButton_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Izhod</string>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>MS Sans Serif</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>true</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="text">
         <string>Opombe</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit_opombe</cstring>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButton_placiloSkupaj</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>radioButton_placiloEnkratno</tabstop>
  <tabstop>lineEdit_TrenutnoPlacilo</tabstop>
  <tabstop>dateEdit</tabstop>
  <tabstop>lineEdit_opombe</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="files.qrc"/>
 </resources>
 <connections/>
</ui>
